packages/qdrant-loader/tests/unit/config/test_chunking_strategy_configs.py

```
@@ -222,6 +223,41 @@ def test_validation_percentage_bounds(self):
+        assert config.max_tokens is None
+        assert config.include_context_in_embed is False
+
+    def test_custom_values(self):
+        """Both knobs can be set explicitly."""
+        config = DoclingStrategyConfig(max_tokens=1024, include_context_in_embed=True)
+        assert config.max_tokens == 1024
+        assert config.include_context_in_embed is True
+
+    def test_max_tokens_must_be_positive(self):
+        """max_tokens is a token budget — zero or negative is meaningless."""
+        with pytest.raises(ValueError):
```

Ruff (0.15.17)

[warning] `pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)